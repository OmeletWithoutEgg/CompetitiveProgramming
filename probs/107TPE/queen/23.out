103540.77 -30769071.13
