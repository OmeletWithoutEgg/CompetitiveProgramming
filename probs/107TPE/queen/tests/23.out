227484.09 -50606186.64
